src/hooks/useMediaQuery/index.ts

```diff
@@ -1,122 +1,122 @@
-import React, { useEffect, useMemo, useState, useCallback, useRef } from "react";
+import React, { useEffect, useMemo, useState, useCallback, useRef } from 'react';
+
 interface IResponsibleHookConfig {
-    [type: string]: string;
+  [type: string]: string;
 }
 
+type Order = 'ASC' | 'DESC';
+
 function checkBrowser() {
-    if (typeof window !== undefined) {
-        return true;
-    }
-    return false;
+  return typeof window !== undefined;
 }
 
 function checkPC() {
-    const PC_OS = "win16|win32|win64|mac|macintel";
-    if (navigator.platform && PC_OS.indexOf(navigator.platform.toLowerCase()) >= 0) {
-        return true;
-    } else {
-        return false;
-    }
+  const PC_OS = 'win16|win32|win64|mac|macintel';
+  return !!(navigator.platform && PC_OS.indexOf(navigator.platform.toLowerCase()) >= 0);
 }
 
-function checkConfig(config: any) {
-    if (!config) {
-        throw new Error("Need to config");
+function checkConfig(config: IResponsibleHookConfig) {
+  if (!config) {
+    throw new Error('Need to config');
+  }
+}
+
+function sortConfigWithSize(config: IResponsibleHookConfig, order: Order = 'ASC') {
+  const configEntries = Object.entries(config);
+  configEntries.sort((before, after) => {
+    const beforeSize = Number(before[1].replace('px', ''));
+    const afterSize = Number(after[1].replace('px', ''));
+    if (!beforeSize || !afterSize || Number.isNaN(beforeSize) || Number.isNaN(afterSize)) {
+      throw new Error('You need to use px in size');
     }
+    if (order === 'ASC') return beforeSize - afterSize;
+    return afterSize - beforeSize;
+  });
+
+  return Object.fromEntries(configEntries);
```

Comment:
🏷 `Object.fromEntries()` 
translate key-value to object like `["key","value"] => { key : value }`